lib/features/modeling/util/ModelingUtil.js

```diff
@@ -37,19 +42,48 @@ export function getParent(element, anyType) {
  * Determines if the local modeling direction is vertical or horizontal.
  *
  * @param {Element} element
+ * @param {ElementRegistry} elementRegistry
  *
  * @return {boolean} false for vertical pools, lanes and their children. true otherwise
  */
-export function isDirectionHorizontal(element) {
+export function isDirectionHorizontal(element, elementRegistry = undefined) {
```

Comment:
Optional argument defaults to `undefined` when not provided:

```suggestion
export function isDirectionHorizontal(element, elementRegistry) {
```